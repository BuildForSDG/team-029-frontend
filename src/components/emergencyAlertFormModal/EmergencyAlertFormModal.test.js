import React from 'react';
import { render, cleanup, fireEvent } from '@testing-library/react';
import EmargencyAlertFormModal from './EmergencyAlertFormModal.jsx';

afterEach(cleanup);

test('Should render <EmergencyAlertFormModal /> component', () => {
	const { asFragment, getByText } = render(<EmargencyAlertFormModal />);
	fireEvent.click(getByText('Send an Alert'));
	expect(asFragment()).toMatchSnapshot();
});
