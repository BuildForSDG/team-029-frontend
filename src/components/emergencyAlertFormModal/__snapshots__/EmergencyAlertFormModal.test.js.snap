// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render <EmergencyAlertFormModal /> component 1`] = `
<DocumentFragment>
  <button
    class="ui button custom-btn transform custom-button"
  >
    Send an Alert
  </button>
</DocumentFragment>
`;
